src/test/java/com/mojang/brigadier/tree/ArgumentCommandNodeTest.java

```diff
@@ -92,7 +92,7 @@ public void testCreateBuilder() throws Exception {
         final RequiredArgumentBuilder<Object, Integer> builder = node.createBuilder();
         assertThat(builder.getName(), is(node.getName()));
         assertThat(builder.getType(), is(node.getType()));
-        assertThat(builder.getRequirement(), is(node.getRequirement()));
+        assertThat(builder.getRequirements(), is(node.getRequirements()));
```

Comment:
For comparing collections, I think you are supposed to use `containsInAnyOrder` instead of `is`.